scripts/generate-f32-vbinary.sh

```diff
@@ -591,4 +591,18 @@ tools/xngen src/f32-vbinary/vopc-avx512f.c.in -D OP=SQRDIFF  -D BATCH_TILE=32 -D
 tools/xngen src/f32-vbinary/vopc-avx512f.c.in -D OP=SUB      -D BATCH_TILE=16 -D ACTIVATION=MINMAX -o src/f32-vbinary/gen/f32-vsubc-minmax-avx512f-u16.c &
 tools/xngen src/f32-vbinary/vopc-avx512f.c.in -D OP=SUB      -D BATCH_TILE=32 -D ACTIVATION=MINMAX -o src/f32-vbinary/gen/f32-vsubc-minmax-avx512f-u32.c &
 
+################################### HEXAGON HVX ##################################
+tools/xngen src/f32-vbinary/vop-hexagon.c.in -D OP=ADD     -D BATCH_TILE=32 -D ACTIVATION=MINMAX -o src/f32-vbinary/gen/f32-vadd-minmax-hexagon-u32.c &
```

Comment:
Based on the other kernels nearby, it seems like the kernel should be suffixed `hvx`. If I understand correctly, `hexagon`:`x86`::`hvx`:`avx512f` (for example).

Reply:
Good point and I will commit after making all related changes.